modules/olm-deleting-metallb-operators-from-a-cluster-using-cli.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * operators/metallb/metallb-upgrading-operator.adoc
+
+:_content-type: PROCEDURE
+[id="olm-deleting-metallb-operator-from-a-cluster-using-cli_{context}"]
+= Deleting MetalLB Operator from a cluster using the CLI
+
+Cluster administrators can delete installed Operators from a selected namespace by using the CLI.
+
+.Prerequisites
+
+- Access to an {product-title} cluster using an account with
```

Comment:
Same here.  This conditional statement needs to be updated. Right now as it is written, the sentence would end with "with" and be a sentence fragment for any repos that are not part of the ifdef statement